### ISSUE TITLE
[Bug] : Version 5.1.6 doesn't build correctly

### ISSUE THREAD

**Issue by stef-corilus** [editorconfig-checker/editorconfig-checker.javascript#413]
Our pipeline installed the 5.1.6 version automatically. However, the installation via npm doesn't go as expected. We get the following error on the npm install (and on the ‘editorconfig-checker -version' command):

```
/usr/local/lib/node_modules/editorconfig-checker/dist/index.js:11
/*! formdata-polyfill. MIT License. Jimmy Wärting <[https://jimmy.warting.se/opensource>
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            

SyntaxError: Unexpected identifier
    at internalCompileFunction (node:internal/vm:76:18)
    at wrapSafe (node:internal/modules/cjs/loader:1283:20)
    at Module._compile (node:internal/modules/cjs/loader:1328:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
    at Module.load (node:internal/modules/cjs/loader:1203:32)
    at Module._load (node:internal/modules/cjs/loader:1019:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
    at node:internal/main/run_main_module:28:49

Node.js v18.20.3
```

I had to shorten the trace, because it was too long.

**Comment by theoludwig** [comment]
Indeed, thank you for the report.
Sorry for the troubles, will revert the changes for now.

**Comment by theoludwig** [comment]
I think it's related to this issue: https://github.com/vercel/ncc/issues/1193

**Comment by theoludwig** [comment]
Ok, I'm very sorry, seems like the issue persists, with the revert of the 2 commits of the release.

In fact, the problematic commit is this one: https://github.com/editorconfig-checker/editorconfig-checker.javascript/commit/d1f6ab15a34db7294ca51ffe8c7590b5a109bfe5.

I reverted the problematic commit, and released v5.1.8, also I improved the CI, so this kind of issue should not happen again. We will directly know if the `@vercel/ncc` build fails.
